releasenotes/notes/features-88f5ec5ec6b5c380.yaml

```diff
@@ -0,0 +1,3 @@
+features:
```

Comment:
is this a feature or a bug fix with how we strip arguments in windows?

Reply:
The previous PR marked it as a feature (probably since it created a new pipeline for stripping arguments for windows), but I will follow Robert's recommendation as I agree that this is more of a bug fix to "account for spaces in the executable path on Windows".